content/90.back-matter.md

```diff
@@ -177,9 +87,7 @@ $$\frac{d(u_i)d(v_j)}{m - d(u_i) - d(v_j) + 1 + d(u_i)d(v_j)}.$$
 Unfortunately, we found that the above edge prior approximation is a poor approximation in many cases.
 We found that the following modified form (introduced in Methods) affords a superior approximation:
 
-\begin{equation}
-    P_{i,j} = \frac{d(u_i) d(v_j)}{\sqrt{(d(u_i) d(v_j))^2 + (m - d(u_i) - d(v_j) + 1)^2}}
-\end{equation}
+$$P_{i,j} = \frac{d(u_i) d(v_j)}{\sqrt{(d(u_i) d(v_j))^2 + (m - d(u_i) - d(v_j) + 1)^2}}$$
```

Comment:
Not sure why, but the previous version rendered the equation in red on my machine. Going to `$$` fixes this.